### DISCUSSION TITLE
Adding destroyBarricade override to BarricadeManager.

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#3744] [Ideas]
Currently, `destroyBarricade`  finds the region based on the x and y bytes and destroys the specified barricade within that region. This is helpful when you don't have the `BarricadeRegion` already, but it seems redundant in the many cases where you already have the region. It would be helpful if there was a `BarricadeManager.destroyBarricade(BarricadeRegion region, BarricadeDrop drop)` method.
(Pssst, we don't have access to `SendDestroyBarricade`, so that's mainly why).

**Comment by SDGNelson** [comment]
Ideally yes in the future plugins should not need to track barricade region at all, it should be an internal implementation detail.

**Comment by MidnightTokyo** [comment]
>we don't have access to SendDestroyBarricade

You don't know this game very well.
You can create your network method variable using CTRL+C CTRL+V from the game code.
![image](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/32614585/6999b854-6bd0-465c-b133-f07be6cfab14)

**Reply by DiFFoZ** [reply]
Yes you can copy and paste RPC method, but if the game will update the method or remove it then your plugin will be broken
